framefileio.cpp

```diff
@@ -33,9 +34,10 @@ bool FrameFileIO::saveFrameFile(QString &fileName, const QVector<CANFrame>* fram
     filters.append(QString(tr("IXXAT MiniLog (*.csv *.CSV)")));
     filters.append(QString(tr("CAN-DO Log (*.can *.avc *.evc *.qcc *.CAN *.AVC *.EVC *.QCC)")));
     filters.append(QString(tr("Vehicle Spy (*.csv *.CSV)")));
-    filters.append(QString(tr("Candump/Kayak(*.log)")));
-    filters.append(QString(tr("Cabana Log(*.csv *.CSV)")));
+    filters.append(QString(tr("Candump/Kayak (*.log)")));
+    filters.append(QString(tr("Cabana Log (*.csv *.CSV)")));
```

Comment:
on osx all listed files are in background color even if the match. Have no idea whats wrong